contents/copy/scholarships/guide.md

```diff
@@ -1,3 +1,114 @@
 ---
-title: Scholarships Guide
+title: Scholarship Holder's Guide
+introText: All of the necessary information you need to know to make your experience at CSSconf EU 2018 a blast!
 ---
+
+## Welcome!
+
+Thank you so much for joining us for CSSconf EU and/or JSConf EU 2018! We're really excited to see you soon in Berlin, but until then, we've created this page to help you with any roadblocks you may be facing on your way there.
+
+## Hotel Information
+
+### [A&O Berlin Friedrichshain](https://www.aohostels.com/en/berlin/berlin-friedrichshain/)
+
+*Boxhagener Str. 73, 10245 Berlin*
+
+We have booked a room for you from:
+- Thursday to Monday for combo ticket holders
```

Comment:
I am _absolutely unsure_ about these times, if anyone could verify them, that'd be cool.

Reply:
Those who got a scholarship with travel grant and let us know in the travel form that they will need a hotel room got a room booked at A&O Hotel Berlin, Köpenicker Str. 44–49. 

No matter which ticket they have.